lib/perl/Genome/VariantReporting/Framework/Component/Expert/Command.pm

```diff
@@ -15,33 +16,114 @@ use Genome::VariantReporting::Framework::FileLookup qw(
 
 class Genome::VariantReporting::Framework::Component::Expert::Command {
     is_abstract => 1,
-    is => ['Genome::Command::DelegatesToResult', 'Genome::VariantReporting::Framework::Component::Base'],
-    has_input => [
+    is => ['Genome::Command::DelegatesToResult', 'Genome::VariantReporting::Framework::Component::WithTranslatedInputs'],
+    attributes_have => {
+        is_planned => {
+            is => "Boolean",
+            default => 0,
+        },
+        is_argument => {
```

Comment:
Let's add a doc to this to explain that these are properties that get passed along to the software result